proto/xudrpc.proto

```diff
@@ -416,6 +439,23 @@ message Balance {
   uint64 unconfirmed_wallet_balance = 6 [json_name = "unconfirmed_wallet_balance"];
 }
 
+message BalanceAlert {
+  // The total balance.
+  uint64 total_balance = 1 [json_name = "total_balance"];
+  // The side of the low balance.
+  enum Side {
+    REMOTE = 0;
+    LOCAL = 1;
+  }
+  Side side = 2 [json_name = "side"];
+  // The bound of the low balance in percentage.
+  uint32 bound = 3 [json_name = "bound"];
+  // The current side balance.
+  uint64 side_balance = 4 [json_name = "side_balance"];
```

Comment:
If we do rename `total_balance` to `capacity` as suggested above, I figure this can just be `balance` instead of `side_balance` since it's clearly implied that it's the balance for this side of the channel.

Reply:
As for total balance, we actually trigger alert for whole trading balance.

Reply:
I'm confused, what exactly is `side_balance` and `total_balance` supposed to represent? If it's not clear to me it definitely won't be clear to people trying to make sense of our API.

Reply:
@raladev @kilrau

Reply:
I think we still need an answer here and I'm not sure what it is.